src/porepy/grids/gmsh/gmsh_interface.py

```diff
@@ -36,8 +36,10 @@ def __init__(self, pts, lines, polygons=None, domain=None, nd=None,
         else:
```

Comment:
Please coordinate with @IvarStefansson , who is also doing some changes to the export filter, see #59